test/jdk/java/awt/Window/MainKeyWindowTest/TestMainKeyWindow.java

```diff
@@ -388,7 +388,7 @@ public static void main(String[] args) throws Exception
             MenuBar mb = new MenuBar();
             mb.add(new Menu("Hello"));
             frame.setMenuBar(mb);
-            frame.setBounds(400, 180, 1068, 821);
+            frame.setBounds(400, 180, 300, 300);
```

Comment:
When a small resolution(800x600) or similar is used, then the window manager moves this big window to the left/top but the test is expected the specific x/y coordinate.